src/app/(proper_react)/redesign/(authenticated)/user/dashboard/View.tsx

```diff
@@ -110,6 +111,8 @@ export const View = (props: Props) => {
       }
     }
 
+    // TODO: Add unit test when changing this code:
+    /* c8 ignore next */
```

Comment:
I snuck this in - because we use faker data, sometimes our unit test cases hit this code path, and sometimes they don't, depending on the shape of generated exposures, thereby sometimes not resulting in 100% "coverage". Down the road we should make our unit tests deterministic and consciously test this code path.